build/yamls/windows/ovs-containerd/conf/Run-AntreaOVS-Containerd.ps1

```diff
@@ -0,0 +1,30 @@
+$ErrorActionPreference = "Stop"
+$OVSInstallDir = "C:\openvswitch"
+$OVS_DB_SCHEMA_PATH = "$OVSInstallDir\usr\share\openvswitch\vswitch.ovsschema"
+$OVS_DB_PATH = "$OVSInstallDir\etc\openvswitch\conf.db"
+if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+    Set-Location -Path "C:\openvswitch\usr\bin
+    ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
+}
+$mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+$mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
+$env:PATH += ";$mountPath/openvswitch/usr/sbin;$mountPath/openvswitch/usr/bin"
+ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+ovs-vsctl --no-wait init
+
+# Set OVS version.
+$OVS_VERSION=$(Get-Item $OVSInstallDir\driver\OVSExt.sys).VersionInfo.ProductVersion
+ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
+
+ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
+
+$SleepInterval = 30
```

Comment:
I feel no need to add the following while-true.

Reply:
The problem then will be how to restart ovsdb-server process if it has stopped ? It has to be started manually then.
Removing this following while-true and running ovs-vswitchd without `--detach` will only take care of ovs-vswitchd process and restart container immediately if ovs-vswitchd process stops.

Reply:
@wenyingd PTAL, thanks.

Reply:
@tnqn This PR contains all changes required for OVS containerisation( except image) and is ready for review.